content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,167 @@
+
+This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is stable, the feature gate is locked and cannot be disabled.
+
+### Prevent unauthorised volume mode conversion during volume restore ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+Blurb for 3141 goes here.
+
+### Pod Scheduling Readiness ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3521 goes here.
+
+### Min domains in PodTopologySpread ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3022 goes here.
+
+### Go workspaces for k/k ([SIG Architecture](https://github.com/kubernetes/community/tree/master/sig-architecture))
+
+Blurb for 4402 goes here.
+
+## Improvements that graduated to beta in Kubernetes v1.30 {#graduations-to-beta}
+
+_This is a selection of some of the improvements that are now beta following the v1.30 release._
+
+### Node log query ([SIG Windows](https://github.com/kubernetes/community/tree/master/sig-windows))
+
+To help with debugging issues on nodes, Kubernetes v1.27 introduced a feature that allows viewing logs of services running on the node. To use the feature, ensure that the NodeLogQuery feature gate is enabled for that node, and that the kubelet configuration options enableSystemLogHandler and enableSystemLogQuery are both set to true. 
+
+On Linux the assumption is that service logs are available via journald. On Windows the assumption is that service logs are available in the application log provider. On both operating systems, logs are also available by reading files within /var/log/.
+[To read more about the feature, please click here](https://kubernetes.io/docs/concepts/cluster-administration/system-logs/#log-query)
+
+### CRD Validation Ratcheting ([SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery))
+
+Blurb for 4008 goes here--subject to Docs Freeze
+
+### Contextual logging ([SIG Instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation))
+
+Blurb for 3077 goes here.
+
+## New alpha features
+
+### Speed up recursive SELinux label change ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+Blurb for 1710 goes here.
+
+### Recursive Read-only (RRO) mounts ([SIG Node](https://github.com/kubernetes/community/tree/master/sig-node))
+
+Blurb for 3857 goes here--subject to Docs Freeze
+
+### Job success/completion policy ([SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps))
+
+Blurb for 3998 goes here--subject to Docs Freeze
```

Comment:
@natalisucks, Did you mean to extend here?

Reply:
yes please @tenzen-y!